addon/components/impagination-dataset.js

```diff
@@ -38,10 +38,14 @@ export default Ember.Component.extend({
 
     let readOffsetAttrFound = get(this, 'read-offset') >= 0;
     Ember.warn('Ember Impagination: `read-offset` attribute has been removed. Please use the `on-init` function instead.', !readOffsetAttrFound, {id: 'ember-impagination.attributes.read-offset'});
-
-    this.get('on-init')(this.get('model'));
+    this.fireOnInit();
```

Comment:
Just making sure: We are not changing any functionality here? Just making it more readable? Are we planning to use `this.fireOnInit()` elsewhere?